Package.swift

```diff
@@ -6,15 +6,19 @@ import PackageDescription
 let package = Package(
     name: "Topsort-Analytics",
     platforms: [
-        .macOS("10.15"),
-        .iOS("13.0"),
+        .macOS("12.00"),
+        .iOS("15.0"),
```

Comment:
What's the reason behind upping the required iOS version? Other SDKs like Segment and Google Ads require iOS 13 and 12

Reply:
I actually lowered the iOS version and see no warnings

Reply:
its because `Date.now` (for sending events), `ProgressView` (loading icon) and `AsyncImage` (show image in banner) are only available on this versions
<img width="1482" alt="image" src="https://github.com/user-attachments/assets/5bb6572a-c0c7-49fd-8576-5125f984f9ca">